src/engine/source/builder/builders/opBuilderHelperActiveResponse.hpp

```diff
@@ -0,0 +1,51 @@
+/* Copyright (C) 2015-2021, Wazuh Inc.
+ * All rights reserved.
+ *
+ */
+
+#ifndef _OP_BUILDER_HELPER_ACTIVE_RESPONSE_H
+#define _OP_BUILDER_HELPER_ACTIVE_RESPONSE_H
+
+#include <any>
+
+#include <baseTypes.hpp>
+
+#include "expression.hpp"
+#include <utils/stringUtils.hpp>
+
+namespace builder::internals::builders
+{
+
+namespace ar
+{
+// TODO: move all the sockets to a shared utils directory
+constexpr const char* AR_QUEUE_PATH {"/var/ossec/queue/alerts/ar"};
+
+constexpr const char* AGENT_ID_PATH {"/agent/id"};
```

Comment:
Add TODO. This should not be hardcoded, when the api is mergeable it can be obtained from `base`.